packages/core/src/directives/template-map.ts

```diff
@@ -67,17 +67,20 @@ const setMapped = (element: Element, name: string, value: unknown, oldValue: unk
       break;
     case MAP_TYPE.PROPERTY:
       if (value === undefined) {
-        delete (element as Element & { [name: string]: typeof value })[name];
+        delete (element as VariableElement<unknown>)[name];
```

Comment:
I you want to leave as Unknwon you can just do

```typescript
type VariableElement = Element & { [name: string]: unknown }
```
```suggestion
        delete (element as VariableElement)[name];
```

Reply:
It is what I meant :)